docs/content/stable/architecture/transactions/read-committed.md

```diff
@@ -1565,7 +1565,10 @@ Read Committed interacts with the following feature:
 
     For more details, see [#12494](https://github.com/yugabyte/yugabyte-db/issues/12494).
 
-* Read restart and serialization errors are not internally handled in read committed isolation if the query's response size exceeds the YB-TServer GFlag `ysql_output_buffer_size`, which has a default value of 256KB (see [#11572](https://github.com/yugabyte/yugabyte-db/issues/11572)).
+* Read restart and serialization errors are not internally handled in read committed isolation:
+- if the query's response size exceeds the YB-TServer GFlag `ysql_output_buffer_size`, which has a default value of 256KB (see [#11572](https://github.com/yugabyte/yugabyte-db/issues/11572)).
```

Comment:
The sub-bullets aren't formatting the way you might want them to. At least in the preview.